rtl/eclrtl/eclhelper_dyn.cpp

```diff
@@ -106,13 +63,14 @@ class ECLRTL_API CDynamicDiskReadArg : public CThorDiskReadArg
         const char *epos = strchr(filter,'=');
         assertex(epos);
```

Comment:
needs a better error message (and also doesn't support ~)

Reply:
Fixed